### PR TITLE
Add BOXBOY!

### PR DESCRIPTION
**Pull request by Gamevita**
Adds BOXBOY!.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/boxboy/game.dat [hunk 1]
```diff
@@ -0,0 +1,20 @@
+title = "BOXBOY!"
+description = "Solve a lot of puzzles in BOXBOY !, available exclusively on the Nintendo eShop of the Nintendo 3DS family consoles! In this 2D platformer, you use blocks to overcome obstacles and reach the end of each stage.\n\nEditor : Nintendo\nGenre : Platforms, Puzzle"
```
Comment (TheKoopaKingdom) [inline games/boxboy/game.dat]: Maybe use a description that's a bit more... neutral? This sounds like it's trying to sell the game.

File: games/boxboy/game.dat [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+needs_system_files = true
+needs_shared_font = true
+game_type = "eshop"
+
+[[releases]]
+title = "0004000000154600"
+region = "tout"
```
Comment (TheKoopaKingdom) [inline games/boxboy/game.dat]: This isn't a valid region.

File: games/boxboy/game.dat [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+release_date = "2015-04-02"
+
+[[testcases]]
+title = "0004000000154500"
+compatibility = "1"
+date = "2017-11-19"
+version = "HEAD-4f13d6f"
+cpu = "AMD Athlon II X2 220"
+gpu = "ATI Radeon HD 4250"
```
Comment (TheKoopaKingdom) [inline games/boxboy/game.dat]: Add AMD here.

File: games/boxboy/game.dat (hunk 1 above)
Comment (Leo626) [inline games/boxboy/game.dat]: Here's the description from Wikipedia:
> BOXBOY! is a puzzle platformer video game developed by HAL Laboratory and published by Nintendo for the Nintendo 3DS handheld console. Players control Qbby, a square-shaped character who can produce a string of connected boxes. The boxes are used to overcome obstacles in stages that Qbby must be guided through.

**Comment by Leo626** [comment]
LGTM pending changes.

**Comment by Leo626** [comment]
Please address the comments or this will be closed.

File: games/boxboy/game.dat (hunk 2 above)
Comment (greggameplayer) [reply games/boxboy/game.dat]: @TheKoopaKingdom that's french, it's mean "all"

There is no way to modify instead of suggest ?

Comment (TheKoopaKingdom) [reply games/boxboy/game.dat]: This is a primarily English project, so we prefer the English spellings for values.

**Comment by TheKoopaKingdom** [comment]
@greggameplayer Do you plan on addressing the review comments? If so I can reopen this PR.

**Comment by greggameplayer** [comment]
@TheKoopaKingdom yes if you want i can modify all the things you've suggest to this game report

**Comment by greggameplayer** [comment]
@TheKoopaKingdom I have one question ,how can i become a member ?

**Comment by TheKoopaKingdom** [comment]
The [Citra Website Team](https://github.com/orgs/citra-emu/teams/website-team/members) is pretty much only moderators, selected off of the criteria of someone actively helping on the forums and Discord server.

**Comment by greggameplayer** [comment]
how can I modify these files ?

**Comment by TheKoopaKingdom** [comment]
Oh my bad, I thought you were the OP, misread the convo. In that case, I'm going to close this, but if you still want to continue this PR, you can follow [this](https://citra-emu.org/wiki/typical-git-workflow/) guide to create a BOXBOY! branch in [your repo](https://github.com/greggameplayer/citra-games-wiki). This guide is for Citra and not the wiki, but everything still applies except for the URLs.

At this point, you should have a new branch based off of the latest master that you can apply the changes from this PR to (Not sure how you'd automatically do that, but copying everything over manually should work.).

**Pull request closed without merging**